.gitignore

```diff
@@ -132,3 +132,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# docker-compose-dev.yml
```

Comment:
should both docker-compose files be excluded?

Reply:
I think that's a good idea. I have excluded both docker-compose files to be excluded.